src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img data-cy="MovieImage" src={movie.imgUrl} alt="Film logo" />
```

Comment:
```suggestion
        <img data-cy="MovieImage" src={imgUrl} alt="Movie logo" />
```

lets change it to 'Movie' for consistency, also lets destructure the 'movie'